src/tasks/node/teardown.js

```diff
@@ -1,18 +1,34 @@
 const path = require('path');
+const { execSync } = require('child_process');
 const chalk = require('chalk');
 const { isYarnAvailable } = require('../../utils');
 
 module.exports = function teardown(config) {
+  const hasYarn = isYarnAvailable();
+  const currentDirectory = process.cwd();
+  const cdPath =
+    path.join(currentDirectory, config.name) === config.path
+      ? config.name
+      : config.path;
+
+  try {
+    const command = hasYarn ? 'yarn' : 'npx';
+
+    execSync(
+      `${command} prettier "${cdPath}/src/**/*.{json,html,css,js,vue,ts,tsx}" --write --config "${cdPath}/.prettierrc"`,
```

Comment:
is the config always available, and does prettier not take that one by default?

Reply:
Yes, the config is available for all our JavaScript templates. If this turns out not to be true, we still swallow the error.

Nope, it takes the "global" one from CISA by default.